src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -25,15 +25,21 @@ public static class Parser
                 // Some Chinese anime releases contain both English and Chinese titles, remove the Chinese title and replace with normal anime pattern
                 new RegexReplace(@"^\[(?:(?<subgroup>[^\]]+?)(?:[\u4E00-\u9FCC]+)?)\]\[(?<title>[^\]]+?)(?:\s(?<chinesetitle>[\u4E00-\u9FCC][^\]]*?))\]\[(?:(?:[\u4E00-\u9FCC]+?)?(?<episode>\d{1,4})(?:[\u4E00-\u9FCC]+?)?)\]", "[${subgroup}] ${title} - ${episode} - ", RegexOptions.Compiled),
 
-                // Chinese LoliHouse/Lilith-Raws releases use Season + Absolute Episode Number, normalize using SxxExx format
+                // Chinese Anime Season Episode Rule 1: Chinese LoliHouse/Lilith-Raws releases use Season - Episode Number format, normalize using SxxExx format
```

Comment:
Remove the `Rule #`. `Chinese Anime with Season and Episode numbers: ...`

Reply:
> How do these releases differ from other anime that uses `Some title S2 - 01` for the release? With XEM aliases those are treated as season 2 aliases with absolute numbers that reset at the beginning of the season, not unlike season/episode numbers.
> 
> I believe @Taloth opened an issue for user provided substitution regexes, but how that would be achieved has not been fleshed out.

Without these 2 new regex rules:
![image](https://user-images.githubusercontent.com/734749/212018857-12ecf81b-8283-46bc-b19f-23583143fa6b.png)
![image](https://user-images.githubusercontent.com/734749/212019058-39c934e6-6b63-4885-878b-be423ebc69d6.png)



The problem is: 
The xem doesn't have `Some title S02` as an alias, the `SubstitutionRegex` is dealing with this situation to convert  `Some title S02` to xem mapped  `Some title S2` title, but for chinese releases, because they are pre-formatted with `PreSubstitutionRegex` to extract the English titles in it, it won't catch by those regex rules that formatting the `S02` to `S2`,  so the relase name will become `Some title S02 - 01` or `Some title S02 [01]` and the 01 is treated as absolute episodes number 01 -> `S1E1`

Just give you an idea how S02 is popular as an naming scheme in chinese releases, I don't know if xem can batch edit every anime to add S02 S03 S04 alias, IMO using regex is less painful than added S0x alias to every animes in xem.
![image](https://user-images.githubusercontent.com/734749/212020586-a284922f-4ab4-4619-aac5-677941c8b133.png)